qiskit/pulse/instructions/play.py

```diff
@@ -18,6 +18,8 @@
 from typing import Optional
 
 from ..channels import PulseChannel
+from ..exceptions import PulseError
+from ..pulse_lib.pulse import Pulse
```

Comment:
`tox -elint` isn't showing any error locally. I'm guessing importing `Pulse` is causing the cyclic import. Can you please try removing `Pulse` imports and type hints?